### ISSUE TITLE
lens Lens Error EACCES permission denied

### ISSUE THREAD

**Issue by Antiarchitect** [NixOS/nixpkgs#261358]
### Describe the bug
Running `lens` from packages shows:

`Could not load extensions: EACCES: permission denied. unlink '/home/andrey/.config/Lens/extensions/cli-lens-extension/cli-lens-extension-0.13.0.tgz'`

### Steps To Reproduce
1. Install lens as non-root user via configuration.nix and `nixos-rebuild switch --upgrade`
2. Run lens as non-root user
3. Get an error window

### Expected behavior
Lens runs without an error above

### Screenshots
![image](https://github.com/NixOS/nixpkgs/assets/50209/4a87270c-a44e-456d-a2a1-dc2a7f427f13)

### Notify maintainers
@dbirks 
@RossComputerGuy 

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.5.6, NixOS, 23.05 (Stoat), 23.05.4172.bd1cde45c778`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.5`
 - channels(root): `"nixos-23.05, nixos-hardware, nixos-unstable"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by teto** [comment]
related https://github.com/NixOS/nixpkgs/issues/123688

**Comment by RossComputerGuy** [comment]
I've ran into this issue before, not relating to nixpkgs I think, the solution was to delete the extensions directory. I think this is a bug with Lens itself and not our packaging.

**Comment by kashw2** [comment]
Can anyone using the nixpkgs version of Lens confirm whether or not this issue is still present?

After https://github.com/NixOS/nixpkgs/pull/242911 this seems to be fixed and if others note the same I'll happily open a PR to backport this to 23.05.

**Comment by kashw2** [comment]
Fixed in https://github.com/NixOS/nixpkgs/pull/242911 which is now available on the 23.11 release channel
